### PR TITLE
WT-12772 Use WT_RET_MSG macros in the block manager when returning EINVAL

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

**Review by korteland — APPROVED** [review]
LGTM - I had a look for the write syscall that Keith mentioned, and `__posix_file_write` uses `WT_RET_MSG` already. So I don't think there's any work to do there.

File: src/block/block_write.c [hunk 1]
```diff
@@ -181,7 +181,9 @@ __wt_block_write_size(WT_SESSION_IMPL *session, WT_BLOCK *block, size_t *sizep)
      * interested in debugging corner cases anyway.
      */
     *sizep = (size_t)WT_ALIGN(*sizep + WT_BLOCK_HEADER_BYTE_SIZE, block->allocsize);
-    return (*sizep > UINT32_MAX - 1024 ? EINVAL : 0);
+    if (*sizep > UINT32_MAX - 1024)
+        WT_RET_MSG(session, EINVAL, "requested block size too large");
```
Comment (keitharnoldsmith) [inline src/block/block_write.c:185]: Grammatical nit: "block size too large" --> "block size is too large"

**Review by keitharnoldsmith — APPROVED** [review]
LGTM -- one small nit that you can fix without a re-review.

File: src/block/block_write.c (hunk 1 above)
Comment (etienneptl) [reply src/block/block_write.c]: Fixed in fb83093f8bb8912a438f7fcc4a987b70034a86b5

**Review by quchenhao — APPROVED** [review]
lgtm

**Pull request merged**